sdk/dotnet/src/Microsoft.Datasync.Client/Offline/SyncContext.cs

```diff
@@ -128,10 +129,39 @@ public async Task InitializeAsync(CancellationToken cancellationToken = default)
                 // Initialize the delta token store.
                 DeltaTokenStore = new DeltaTokenStore(OfflineStore);
 
+                // Clear the errors in the store.
+                var errorsEnumerable = new FuncAsyncPageable<TableOperationError>(nextLink => GetPageOfErrorsAsync("", nextLink, cancellationToken));
+                var errors = await errorsEnumerable.ToZumoListAsync(cancellationToken).ConfigureAwait(false);
+                //await RemoveErrorsAsync(errors, cancellationToken).ConfigureAwait(false);
```

Comment:
Is the commented line 135 correct? There is no "Clear the errors..." execution! I was wondering why an error is still stuck in __error after initialization.

Reply:
Ouch (for me) - yes, you are right.  I'll get a fix in this weekend (separately)

Reply:
Fixed in 5.1.1

Reply:
Thanks for this very quick fix!